modbusApp/src/modbusInterpose.c

```diff
@@ -474,25 +476,28 @@ static asynStatus getOutputEos(void *ppvt, asynUser *pasynUser,
                                      eos, eossize, eoslen);
 }
 
-
+
 /* register modbusInterposeConfig*/
 static const iocshArg modbusInterposeConfigArg0 = { "portName", iocshArgString };
 static const iocshArg modbusInterposeConfigArg1 = { "link type", iocshArgInt };
 static const iocshArg modbusInterposeConfigArg2 = { "timeout (msec)", iocshArgInt };
 static const iocshArg modbusInterposeConfigArg3 = { "write delay (msec)", iocshArgInt };
+static const iocshArg modbusInterposeConfigArg4 = { "skip transaction id", iocshArgInt };
```

Comment:
Do we want to call this "skip transaction id _check_" maybe?

Skip transaction ID sounds like it might be about skipping sending a particular transaction ID.

Reply:
yes, i was thinking that